### PR TITLE
Added test scheduler & the starter files of the dashboard.

### PR DESCRIPTION
**Pull request by theakhiljha**
This pull request contains the initial files of the shiny dashboard app using golem and the automated testing files which are triggered:

-  At 0605 UTC
-  Push and Pull Request

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -37,3 +37,4 @@ vignettes/*.pdf
 
 # R Environment Variables
 .Renviron
+>>>>>>> 7c51ef655768a775f805bfbdef962df82eeda442
```
Comment (mdietze) [inline .gitignore:40]: merge conflict that needs to be resolved

File: DESCRIPTION [hunk 2]
```diff
@@ -0,0 +1,26 @@
+Package: statusboard
+Title: PEcAn Status Board
+Version: 0.0.0.9000
+Authors@R: person('AUTHOR_FIRST', 'AUTHOR_LAST', email = 'AUTHOR@MAIL.COM', role = c('cre', 'aut'))
+Description: PKG_DESC.
+License: MIT + file LICENSE
```
Comment (mdietze) [inline DESCRIPTION:6]: Author and description needs to be filled in. For license we've used NCSA instead of MIT

File: dev/01_start.R [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  author_email = "AUTHOR@MAIL.COM", # Your Email
+  repo_url = NULL # The URL of the GitHub Repo (optional)
+)     
+
+## Set {golem} options ----
+golem::set_golem_options()
+
+## Create Common Files ----
+## See ?usethis for more information
+usethis::use_mit_license( "Golem User" )  # You can set another license here
```
Comment (mdietze) [inline dev/01_start.R:36]: update license

File: dev/02_dev.R [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+# Circle CI
+usethis::use_circleci()
+usethis::use_circleci_badge()
+
+# Jenkins
+usethis::use_jenkins()
+
+# GitLab CI
+usethis::use_gitlab_ci()
```
Comment (mdietze) [inline dev/02_dev.R:98]: Not clear to me why the app would depend on Travis and Jenkins, which we haven't used in years. Also, what are AppVeyor, Circle CI, and GitLab CI and what are they being used for.

File: inst/run-test-list.R [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+stages$success_status <-
+  ifelse(grepl("DONE", stages$stage), TRUE, FALSE)
+models_name <- search.models(server)
+sites_name <- search.sites(server)
+stages$model_name <-
+  models_name$models$model_name[match(stages$model_id, models_name$models$model_id)]
+stages$site_name <-
+  sites_name$sites$sitename[match(stages$site_id, sites_name$sites$id)]
+
+saveRDS(stages, file = "inst/test_result.rds")
```
Comment (mdietze) [inline inst/run-test-list.R:93]: Is it a good idea to save the results as a binary? That will change every day but can't be diff'ed. Seems like a csv would be easier to append to.

Comment (theakhiljha) [reply inst/run-test-list.R]: Updated, Now the generated result is being saved as a CSV file.

**Comment by theakhiljha** [comment]
Merged in Pull request #2.

**Pull request closed without merging**